docs/copy-docs.py

```diff
@@ -203,12 +206,14 @@ def update_toc(toc, paths, ignore=[]):
         files = [section['file'] for section in sections]
 
         # find and remove deleted files from toc
-        for i, path in enumerate(files):
+        j = 0
+        for path in files:
             if path in remaining_paths:
                 remaining_paths.remove(path)
+                j += 1
             else:
                 print(f'deleting {path} from toc')
-                del sections[i]  # delete from toc
+                del sections[j]  # delete from toc
```

Comment:
Previously, this was causing the list to be changed while iterating over it, which made some documents disappear from the toc.

Reply:
😬